CheckboxesValidation/Student.java

```diff
@@ -0,0 +1,57 @@
+package SpringMVCBasics;
+
+import java.util.LinkedHashMap;
+
+public class Student {
```

Comment:
Code is good and well structured. A small suggestion is to use Lombok Library for avoiding code that is too verbose with  constructors, getters, setters

Reply:
Okay